handbook/engineering/distribution/goals.md

```diff
@@ -93,6 +93,31 @@ We will initially focus on reducing the time it takes to collect troubleshooting
   - ~~Create code to bootstrap new projects.~~ Done: 3.21
   - Document project and folder usage guidelines.
 
+### Site admins use reliable methods like Docker Compose or Kubernetes for production deployments
```

Comment:
I copied this over from https://github.com/sourcegraph/about/pull/1104/files - I think this is a valuable goal worth calling out still here (even if it turns out to be something I work on)